vme/src/modify.cpp

```diff
@@ -1056,30 +1045,29 @@ void do_set(unit_data *ch, char *argument, const command_info *cmd)
             else
             {
                 slog(LOG_ALL, 0, "PASSWORD: %s changed %s's password.", UNIT_NAME(ch), UNIT_NAME(unt));
-                strncpy(PC_PWD(unt), crypt(strarg, PC_FILENAME(unt)), PC_MAX_PASSWORD);
-                PC_PWD(unt)[PC_MAX_PASSWORD - 1] = 0;
+                UPC(unt)->setPassword(crypt(strarg, PC_FILENAME(unt)));
```

Comment:
Double check that you chop it

Reply:
Yes, it is done in the setter

Reply:
Agreed. It was a comment to myself but didn't go as intended :-)